### DISCUSSION TITLE
Interest in GSoC 23

### DISCUSSION THREAD

**Discussion by osamamagdy** [OWASP/wrongsecrets#611] [Q&A] [answered]
Hello everyone
I am Osama Magdy, a final-year student at the Computer Engineering Department - at Cairo University, Egypt. I am graduating next June with a two-year DevOps working experience. I also have experience with open source contribution through my project with GSoC 2022 which contributed to supply chain security and introduced me to many concepts about the software's cryptography and security. This also increased my skills in working remotely with international teams from different cultures. I can work with a vast set of tools and programming languages in the cloud and infrastructure field which includes:
Languages: Golang, C/C++, JavaScript, Python, SQL
Cloud Computing & DevOps Tools: AWS, Docker, Kubernetes, Jenkins, ArgoCD, Helm, Terraform, Prometheus and Grafana
Web Development: HTML5, CSS3, Node.js, Flask
I am really excited and interested in contributing this Year with wrongsecrets and there are multiple ideas that caught my interest and are aligned with my experience. I commented on those issues https://github.com/OWASP/wrongsecrets-ctf-party/issues/156, https://github.com/OWASP/wrongsecrets-ctf-party/issues/86, https://github.com/OWASP/wrongsecrets-ctf-party/issues/84 and waiting for a response. 
I also tried to join the slack community with my email osamamagdy174@gmail.com but with no luck 🤔 so I'm asking if it's possible to invite me there.

**Comment by commjoen** [comment]
Hi @osamamagdy , what happens if you register with your e-mailadres at https://owasp.org/slack/invite ?

**Reply by commjoen** [reply]
I got an email not about OWASP, but from Harold Blankenship. Do you get that one in your inbox as well?

**Reply by osamamagdy** [reply]
It only said `something went wrong or you already have an account there` and I didn't receive any email. Thanks to @bendehaan, I got an invitation now 🙏

**Comment by bendehaan** [comment, marked as answer]
@osamamagdy sent you an invite on the e-mail you mentioned. Does that work?

**Reply by osamamagdy** [reply]
Yes, it does. Thanks 🙌
